### DISCUSSION TITLE
✔️ Github things

### DISCUSSION THREAD

**Discussion by kevingrillet** [zebscripts/AFK-Daily#62] [Ideas]
- Maybe hide Actions, Projects & Security tabs, which are not used here.
- Create the wiki #27 (I don't know if others can update/PR the wiki), 
- Then redo the `readme.md` with funky style
   - Animated `.gif` console
   - https://shields.io/ for license, techno, requirements, ... (more images, less text) [[Examples](https://github.com/alexandresanlim/Badges4-README.md-Profile)]

[![License](https://img.shields.io/badge/License-MIT-darkgrey)](https://shields.io/)
[![Windows 10](https://img.shields.io/badge/Windows-10-darkblue?logo=windows)](https://shields.io/)
[![Visual Studio Code](https://img.shields.io/badge/Editor-Visual%20Studio%20Code-darkblue?logo=visual-studio-code)](https://shields.io/)
[![Bash](https://img.shields.io/badge/Shell-Bash-green?logo=gnu-bash)](https://shields.io/)
[![Bluestacks](https://img.shields.io/badge/Emulator-Bluestacks-darkgreen)](https://shields.io/)
[![Nox](https://img.shields.io/badge/Emulator-Nox-blue)](https://shields.io/)

   - https://github.com/anuraghazra/github-readme-stats#github-extra-pins

[![Readme Card](https://github-readme-stats.vercel.app/api/pin/?username=zebscripts&repo=AFK-Daily&theme=darcula)](https://github.com/anuraghazra/github-readme-stats)

   - https://github.com/kyechan99/capsule-render for header/footer

![header](https://capsule-render.vercel.app/api?type=waving&color=gradient&height=300&section=header&text=AFK-Daily&fontSize=90)

![footer](https://capsule-render.vercel.app/api?section=footer&type=waving&color=gradient&height=100)

**Comment by Zebiano** [comment]
Cool! I'm always down to make things more appealing and organised. Thank you for all the suggestions!

> Maybe hide Actions, Projects & Security tabs, which are not used here.

Actions and Projects tabs are now hidden, though I did not manage to change the visibility for the Security Tab. That's OK though.

> Wiki

Created. Let me know if you can edit it or not!

Regarding the `README.md` changes, I think a GIF showing console logs would be a bit boring honestly... I could try and make a gif of the game auto-doing stuff, but I'm unsure if I *actually* want to do that. The shields are always cool and a nice addition, I might add some of them. Though I don't like to add unnecessary badges as well.

> Extra pins

I'm a bit confused. Are you showing me this so I can add it to my profile or add it to the `README.md`? On my profile, I only have 4 pins for now, so that's not a problem, and I don't think I'll want to add that to the `README.md`.

> Header/Footer

Uuuh, fancy! I like it. Will look into it, thank you!

Any other ideas are welcome as well!

**Reply by kevingrillet** [reply]
> I think a GIF showing console logs would be a bit boring honestly...

Maybe a speed up [video](https://github.blog/changelog/2020-12-16-video-upload-public-beta/) of the console + emulator?

**Reply by Zebiano** [reply]
Not a big fan of it because I'd like to keep my AFK account somewhat under the radar. But there are parts of the script where the account is not shown, so technically that could work. Also, whenever changes are made, the `.gif` would have to be updated, which I can already tell you now I'll be too lazy to do. 😅 

Though feel free to make a `.gif` and I can see if it's worth adding it!

**Comment by kevingrillet** [comment]
> I did not manage to change the visibility for the Security Tab.

Oh, I thought it was possible to hide the Security tab, but no 😢

> Created. Let me know if you can edit it or not!

We can't edit the wiki 😭

A possible solution, but boring... solution found [on redit](https://stackoverflow.com/a/11481887/7295428) with the [broken link in the post](https://web.archive.org/web/20160307192232/http://fusiongrokker.com/post/how-you-can-contribute-to-taffy-documentation).

Or just let "everyone" edit the wiki, or add us to the collaborators [*](https://docs.github.com/en/communities/documenting-your-project-with-wikis/changing-access-permissions-for-wikis)

> Extra pins

I did not really look into it, but having stats on the repo could be good looking (see issues closed, PR, contributions, commits,...) but it's not what it does, it has been just the concept behind 😅

**Reply by Zebiano** [reply]
For now, I've made the Wiki publicly editable. I'll keep an eye on it to check if unwanted behaviours occur or not, but I'm positive people will be civil about it.

**Reply by kevingrillet** [reply]
I've started to copy/paste things from the `readme.md` to the wiki + some adds.

I think it could be more practical if we move to a structure like [that](https://github.com/WebTools-NG/WebTools-NG/wiki).

**Reply by Zebiano** [reply]
I like it! Feel free to create something of that sorts, I would eventually build something like it anyways.

Also, I've changed some permissions, let me know if you can still edit the Wiki.

**Reply by kevingrillet** [reply]
Quick update, tell me if you want to see other changes.

**Reply by Zebiano** [reply]
Sorry for keeping quiet. If you allow me, I'd like to take care of this one. I'll see what I can do with the suggestions you made, and I'll let you know when I'm done so you can check them out and tell me your opinion!
